### DISCUSSION TITLE
Is finding/replacing images possible?

### DISCUSSION THREAD

**Discussion by johnmcmillan** [Hopding/pdf-lib#1623] [General]
I apologize if there's an easy to follow article that I've missed. I've run all of the usual suspect searches with no luck. I'm using node.js and I'm trying to open a PDF and replaces some images. Any help would be greatly appreciated.

**Comment by jacobkapitein** [comment]
Yes, it is possible.
In this issue is a comment with a zip file that has helped me a lot: https://github.com/Hopding/pdf-lib/issues/175
